config/locales/en.yml

```
@@ -249,7 +249,8 @@ en:
           actions: "Actions"
         dossier_action:
           edit_dossier: "Edit the file"
-          start_other_dossier: "Start an other file"
+          start_other_dossier: "Start an other empty file"
+          clone: "Clone this file"
```

```suggestion
          clone: "Duplicate file"
```

Je pense que "clone" est un peu trop technique – on s'adresse à des humains ici ;)